### PR TITLE
This commit adds a README.md for openshift-ci usage

### PR DESCRIPTION
**Pull request by tonyxrmdavidson**
This commit adds a README.md for openshift-ci usage explaining how to locally run the script and how to start the associated openshift-ci job.

### CODE REVIEW

File: openshift-ci/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+From the root of the repository
+```shell
+./openshift-ci/scripts/oc-model-registry.-deploy.sh
+```
+
+## Runing the openshift-ci
+
+You can start the openshift-ci job to test changes in your Pull Request. To do so put the following command into a comment in your Pull Request
+```shell
```
Comment (tarilabs) [inline openshift-ci/README.md:29]:
```suggestion
```
```

File: openshift-ci/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Pre-requisites:
+
+You will need to have an openshift cluster deployed and be oc logged into you cluster as admin.
+
+## Runing the script:
+
+From the root of the repository
+```shell
+./openshift-ci/scripts/oc-model-registry.-deploy.sh
```
Comment (tarilabs) [inline openshift-ci/README.md]:
```suggestion
./openshift-ci/scripts/oc-model-registry-deploy.sh
```

File: openshift-ci/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Model Registry Deployment and Deployment Test
+
+This deployment of model-registry is deployed via Opendatahub and used the ODH nightly images deployed to an openshfit cluster.
```
Comment (tarilabs) [inline openshift-ci/README.md:3]: consider putting an hyperlink to the OpenShift CI "evironment"?
Somewhere where we could see previous runs or the likes?

This way people not familiar yet with OpenShift CI could better understand which CI system is being discussed here

**Review by tarilabs — COMMENTED** [review]
minor comment, thank you

File: openshift-ci/README.md (hunk 1 above)
Comment (tarilabs) [inline openshift-ci/README.md]: consider avoiding `shell` in ` ```shell `

**Review by tarilabs — COMMENTED** [review]
minor comment, thank you

File: openshift-ci/README.md (hunk 3 above)
Comment (tonyxrmdavidson) [reply openshift-ci/README.md:3]: Good suggestion, done

File: openshift-ci/README.md (hunk 1 above)
Comment (tonyxrmdavidson) [reply openshift-ci/README.md]: Done

**Review by tarilabs — APPROVED** [review]
/lgtm
/approve

**Pull request merged**